### ISSUE TITLE
Converting from mkv to mp4 ruins audio on macOS 11.3

### ISSUE THREAD

**Issue by GimmyTomas** [obsproject/obs-studio#4626]
### Operating System Info

Other

### Other OS

macOS 11.3

### OBS Studio Version

26.1.2

### OBS Studio Log URL

https://obsproject.com/logs/nywHre9bfOheFOdp

### Expected Behavior

Converting (with OBS) the mkv file to mp4 should not alter the audio quality.

### Current Behavior

Converting (with OBS) the mkv file to mp4 alters the audio quality, introducing various distortions and noises. If converting the same mkv file with an online converter, the output mp4 has the same audio quality as the mkv, as it should be. Therefore, I expect this to be an OBS issue. Before updating macOS to 11.3, OBS converted mkv files with no issue.

### Steps to Reproduce

1. Update to macOS 11.3.
2. Record a video and save it as mkv.
3. Use OBS to convert the mkv file to mp4.
4. Compare the audio of the two videos.

**Comment by RytoEX** [comment]
How are you using OBS to convert an MKV to an MP4?  What online converter did you use?  How are you comparing the audio of the two videos?  You're saying this all worked fine on macOS 11.2?

**Comment by GimmyTomas** [comment]
In OBS, from "File > Converti le registrazioni" (my computer is in Italian, I suppose it is "File > Convert the videos" in English), I select the mkv file produced by OBS, choose the new name (including the .mp4 extension) and click "convert".

I used this online converter https://cloudconvert.com/mkv-to-mp4.

The audio is very very clearly distorted, noisy and lower quality with respect to both the mp4 from the online converter and the mkv. I did not use any particular technique to compare them, since it is extremely clear that something is wrong with the audio of the mp4 produced by OBS.

Yes, I used to do this conversion often (almost daily) and it always worked fine. It changed behavior right after the update to 11.3, so I suppose that could be the cause.

**Comment by RytoEX** [comment]
What are you using to open the files to compare the audio?  What software, what version?

**Comment by GimmyTomas** [comment]
QuickTime Player 10.5 (1086.4.2).

The audio still looked the same after uploading the mp4 to youtube though.

**Comment by RytoEX** [comment]
> QuickTime Player 10.5 (1086.4.2).

Check in VLC.

> The audio still looked the same after uploading the mp4 to youtube though.

Looked the same as what?  The good audio?  Or the bad audio?

**Comment by GimmyTomas** [comment]
You are right, VLC plays the mp4 (output of the OBS conversion) without audio distortions, therefore the issue is probably with QuickTime Player and not with OBS.

It may be worth noting that QuickTime Player plays other mp4 files well, and seems to have problems only with the files produced by OBS. For example, as I said, QuickTime plays the mp4 produced by the online converter better. In case this is not considered to be relevant for OBS, I guess that the issue may be closed. Thank you!

(For youtube, nevermind, I was wrong. I meant that it looked as the bad audio, but I judged on a very short video with bad/low audio quality. I just did a test, and on youtube the mp4 produced by OBS plays well.)
